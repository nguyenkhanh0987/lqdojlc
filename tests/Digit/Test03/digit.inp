100
58810 99545
4096 61960
13703 55932
87764 92827
44988 78556
8763 33478
39199 60909
89143 93020
29938 88115
27262 77298
9154 30020
45576 72383
84449 99615
6577 83610
21867 89718
45556 93205
32555 48324
88603 97302
15358 49966
47644 53061
28592 72198
8879 17769
78849 97657
79460 98298
95372 99525
9645 46967
85881 97447
54885 58576
75829 90244
59585 70102
93054 95379
19766 67824
32077 62079
8117 15852
47221 84014
53144 86818
55969 86607
57656 62786
59596 85242
27139 87651
53458 83153
1002 51076
80744 98528
51366 78748
24178 93370
47041 96440
27810 60899
25959 92375
27653 28556
76018 98575
96361 98598
79604 84375
36833 68639
26502 88250
54194 66184
40594 78457
54054 86823
53999 77513
72368 96428
44489 72254
85091 95081
67058 75678
96790 96878
94557 95430
74659 84760
47828 64145
65915 71673
88494 96517
82880 84416
66432 94593
72600 95083
25776 80437
11284 47261
20987 69794
25019 43099
17195 48814
56428 63044
92307 96350
23988 37062
57944 63878
78608 94886
54307 66977
13386 90353
58163 67541
99730 99756
16687 86919
11445 67327
99012 99242
16568 38140
52766 87818
3136 76088
5546 75104
18916 94894
88020 99004
7459 51155
36376 56641
96675 96848
44370 66946
47237 72310
68228 98537
