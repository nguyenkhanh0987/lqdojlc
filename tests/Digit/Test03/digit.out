2008
8709
5977
198
2808
4219
2221
81
4793
5041
3899
2380
575
9273
6507
3240
2034
209
5424
543
4476
1652
608
548
210
5594
400
485
764
727
29
5481
3338
1408
2658
2110
1935
437
1277
5531
1994
8287
521
1904
6268
3121
4009
5996
241
924
108
47
3247
5685
961
3016
2077
1715
910
2489
340
525
0
17
477
1429
524
180
26
1331
871
5644
5456
5699
2708
4753
592
54
2034
449
528
996
8209
682
0
7305
7204
43
3554
2187
9665
9100
6973
361
6951
2278
0
2055
2150
1288
